parquet-hadoop/src/main/java/org/apache/parquet/hadoop/ParquetFileWriter.java

```diff
@@ -484,7 +484,8 @@ private static void serializeFooter(ParquetMetadata footer, FSDataOutputStream o
    * @throws IOException
    */
   public static void writeMetadataFile(Configuration configuration, Path outputPath, List<Footer> footers) throws IOException {
-    ParquetMetadata metadataFooter = mergeFooters(outputPath, footers);
+    Path qualifiedPath = FileSystem.get(configuration).makeQualified(outputPath);
```

Comment:
Won't this throw an exception if the outputPath is in a FS other than the default?